### PR TITLE
[Update] links to information views

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Based on the discussion in: https://www.timescale.com/forum/t/get-compression-policy/286
"I see the functions “add_compression_policy” and “remove_compression_policy”. What do I use to check current existing compression policy or policies?"

Added links to informational-view in relevant sections.
# Links

Fixes #[insert issue link, if any]

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

### CODE REVIEW

File: api/actions.md [hunk 1]
```diff
@@ -7,4 +7,8 @@ even enhancing existing policies with additional functionality.
 
 The following APIs and views allow you to manage the jobs that you create and
 get details around automatic jobs used by other TimescaleDB functions like
-Continuous Aggregation refresh policies and data retention policies.
+Continuous Aggregation refresh policies and data retention policies. To view the
+policies that you set or the policies that already exist, see [informational
+views][informational-views].
```
Comment (Loquacity) [inline api/actions.md:12]:
```suggestion
continuous aggregation refresh policies and data retention policies. To view the
policies that you set or the policies that already exist, see 
[informational views][informational-views].
```

File: api/add_compression_policy.md [hunk 2]
```diff
@@ -3,10 +3,12 @@ Allows you to set a policy by which the system compresses a chunk
...
-aggregates that already have compression enabled. Use the [`ALTER TABLE`][compression_alter-table] command
-to set `timescaledb.compress` and other configuration parameters for hypertables.
-Use [`ALTER MATERIALIZED VIEW`][compression_continuous-aggregate] command to
-enable compression on continuous aggregates
+aggregates that already have compression enabled. Use the [`ALTER
+TABLE`][compression_alter-table] command to set `timescaledb.compress` and other
+configuration parameters for hypertables. Use [`ALTER MATERIALIZED
+VIEW`][compression_continuous-aggregate] command to enable compression on
+continuous aggregates. To view the policies that you set or the policies that
+already exist, see [informational views][informational-views].
```
Comment (Loquacity) [inline api/add_compression_policy.md:11]:
```suggestion
aggregates that already have compression enabled. Use the 
[`ALTER TABLE`][compression_alter-table] command to set `timescaledb.compress` 
and other configuration parameters for hypertables. Use 
[`ALTER MATERIALIZED VIEW`][compression_continuous-aggregate] command to 
enable compression on continuous aggregates. To view the policies that you set or 
the policies that already exist, see 
[informational views][informational-views].
```

File: api/add_continuous_aggregate_policy.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ## add_continuous_aggregate_policy() <tag type="community">Community</tag>
-Create a policy that automatically refreshes a continuous aggregate.
+Create a policy that automatically refreshes a continuous aggregate. To view the
+policies that you set or the policies that already exist, see [informational
+views][informational-views].
```
Comment (Loquacity) [inline api/add_continuous_aggregate_policy.md:4]:
```suggestion
policies that you set or the policies that already exist, see 
[informational views][informational-views].
```

File: api/remove_compression_policy.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 ## remove_compression_policy() <tag type="community" content="community" />
-If you need to remove the compression policy. To re-start policy-based compression again you need to re-add the policy.
+If you need to remove the compression policy. To re-start policy-based
```
Comment (Loquacity) [inline api/remove_compression_policy.md:2]:
```suggestion
If you need to remove the compression policy. To restart policy-based
```

File: api/remove_compression_policy.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
...
+compression again you need to re-add the policy. To view the policies that
```
Comment (Loquacity) [inline api/remove_compression_policy.md:3]:
```suggestion
compression you need to add the policy again. To view the policies that
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Review by Loquacity — APPROVED** [review]

**Review by mirandaauhl — APPROVED** [review]
Looks great!

**Pull request merged**